torch/_export/serde/serialize.py

```diff
@@ -624,10 +668,15 @@ def serialize(self, exported_program: ep.ExportedProgram) -> Tuple[ExportedProgr
                 exported_program.call_spec
             ).serialize(exported_program.graph_module)
         )
+        serialized_range_constraints = serialize_range_constraints(exported_program.range_constraints)
+        serialized_equality_constraints = serialize_equality_constraints(exported_program.equality_constraints)
+
         return (
             ExportedProgram(
                 graph_module=serialized_graph_module,
-                opset_version=self.opset_version
```

Comment:
how did this pass lint, lol...don't we insist on trailing commas?

Reply:
I don't think the OSS linter insists on having trailing commas